src/utils.rs

```diff
@@ -55,25 +154,34 @@ pub fn convert_data_type(data_type: &str) -> String {
         let vec_type = format!("Vec<{}>", array_of_type);
         return vec_type;
     }
+    let state = STATE.get().unwrap();
 
     match data_type {
         "bool" | "boolean" => "bool",
         "bytea" => "Vec<u8>", // is this right?
         "char" | "bpchar" | "character" => "String",
-        "date" => "chrono::NaiveDate",
+        "date" => state.date_time_lib.date_type(),
         "float4" | "real" => "f32",
-        "float8" | "double precision" => "f64",
+        "float8" | "double precision" | "numeric" => "f64",
         "int2" | "smallint" | "smallserial" => "i16",
         "int4" | "int" | "serial" => "i32",
         "int8" | "bigint" | "bigserial" => "i64",
         "void" => "()",
         "jsonb" | "json" => "serde_json::Value",
-        "text" | "varchar" | "name" | "citext" => "String",
-        "time" => "chrono::NaiveTime",
-        "timestamp" => "chrono::NaiveDateTime",
-        "timestamptz" => "chrono::DateTime<chrono::Utc>",
+        "text" | "_text" | "varchar" | "name" | "citext" => "String",
```

Comment:
is `_text` not going to be an array of strings? Most _ prefixes are

Reply:
nice catch. it was panicking on me for the `_text`. I forgot to remove it :)